types/database/schema/ChannelList.types.ts

```diff
@@ -2,6 +2,7 @@
 import {SQLiteDatabase} from 'react-native-sqlite-storage';
 
 import {ChatListDetail} from './ChatListDetail.types';
+import {MessageType} from '../../../src/hooks/core/websocket/types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `MessageType` seems to be incorrect. It should be relative to the current file, not from `src`. Please correct the import path.
```diff
- import {MessageType} from '../../../src/hooks/core/websocket/types';
+ import {MessageType} from '../../hooks/core/websocket/types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->